src/app/[locale]/projects/detail/[id]/components/ProjectVulnerabilities.tsx

```diff
@@ -17,6 +17,24 @@ import { useEffect, useState } from 'react'
 import { Tab, Tabs } from 'react-bootstrap'
 import VulnerabilityTab from './VulnerabilityTab'
 
+const extractLinkedProjects = (data: any, d: ProjectData[]) => {
```

Comment:
Avoid using **any**

Reply:
Please give variable a meaningful name, example **d** -> **projectsData**. It will help other people understand the code more easily

Reply:
Done